### PR TITLE
StudentExamFee AcceptanceTests

### PR DESCRIPTION
**Pull request by viralpandya**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Acceptance/APIs/StudentExamFees/StudentExamFeesApiTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            StudentExamFee randomStudentExamFee = await PostRandomStudentExamFeeAsync();
+            StudentExamFee modifiedStudentExamFee = await UpdateStudentExamFeeRandom(randomStudentExamFee);
+
+            // when
+            await this.otripleSApiBroker.PutStudentExamFeeAsync(modifiedStudentExamFee);
+
+            StudentExamFee actualStudentExamFee =
+                await this.otripleSApiBroker.GetStudentExamFeeByIdsAsync(
+                   randomStudentExamFee.StudentId,
+                   randomStudentExamFee.ExamFeeId);
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Acceptance/APIs/StudentExamFees/StudentExamFeesApiTests.Logic.cs:52]: Please consider using variable names for inputs

**Comment by hassanhabib** [comment]
@viralpandya one small comment and please address conflicts

File: OtripleS.Web.Api.Tests.Acceptance/APIs/StudentExamFees/StudentExamFeesApiTests.Logic.cs (hunk 1 above)
Comment (viralpandya) [reply OtripleS.Web.Api.Tests.Acceptance/APIs/StudentExamFees/StudentExamFeesApiTests.Logic.cs:52]: working on it now

Comment (viralpandya) [reply OtripleS.Web.Api.Tests.Acceptance/APIs/StudentExamFees/StudentExamFeesApiTests.Logic.cs]: Resolved

**Pull request merged**